jobs/transforms/base_model_audits_test.py

```diff
@@ -19,6 +19,8 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import re
+from core.domain import change_domain
```

Comment:
```suggestion
import re

from core.domain import change_domain
```
Also, this should have been caught by the lint checks. cc @DubeySandeep Are sort lint checks enabled for jobs?

Reply:
@vojtechjelinek the link checks are enabled I think because it gave me lint error when I imported it in the wrong place but it did not warn me about new line

Reply:
Looks like it isn't working for this new dir, I'm looking into it and will give an update!

Reply:
@YBCS Can you change this?

Reply:
Done

Reply:
@YBCS Can you please create an issue for the lint issue and assign it to me? I need to take a deeper look and it will take time to fix this issue. [I'll prioritize fixing this issue and the GitHub issue will help us track this task.]

Reply:
@DubeySandeep Done  PTAL!
https://github.com/oppia/oppia/issues/12786